python_Docker_build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9
```

Comment:
This python docker image is for running ANY python script that only depends on pandas or Bill's python scripts (from https://github.com/tncowart/python), not just bird. 

I think the docker file should download Bill's script, add their location to the PYTHON_PATH (I think) env var, install pandas and then the ENTRYPOINT should be ["python"].

Reply:
@brendan-kearney Your changes haven't addressed my issues. It still doesn't download Bill's scripts from https://github.com/tncowart/python and add the directory they are in to PYTHON_PATH. The entry point is also still ["python", "bird.py"] and I think it should just be ["python"]